executor/write.go

```diff
@@ -180,10 +181,11 @@ func updateRecord(ctx sessionctx.Context, h int64, oldData, newData []types.Datu
 
 // resetErrDataTooLong reset ErrDataTooLong error msg.
 // types.ErrDataTooLong is produced in types.ProduceStrWithSpecifiedTp, there is no column info in there,
-// so we reset the error msg here, and wrap old err with errors.Wrap.
+// so we reset the error msg here, and wrap old err with new error.
```

Comment:
update the comment?

Reply:
yes, it's no longer use errors.Wrap so change the comment